src/main/java/com/example/nasastealphoto/model/Picture.java

```diff
@@ -0,0 +1,38 @@
+package com.example.nasastealphoto.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import java.time.LocalDateTime;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "pictures")
+public class Picture {
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "picture_id_gen")
+    @SequenceGenerator(name = "picture_id_gen", sequenceName = "pictures_seq", allocationSize = 1)
+    private Long id;
+
+    @Column(nullable = false, unique = true)
+    private Long nasaId;
+
+    @Column(nullable = false)
+    private  String imgSrc;
+
+    @CreationTimestamp
+    private LocalDateTime createdAt;
+
+    @ManyToOne(fetch = FetchType.EAGER, optional = false)
```

Comment:
@romanovosad87 EAGER is a default fetch type for `*toOne` relationships.

Reply:
When setting annotation properties, you should make a habit of looking inside and checking the default value.